ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -80,9 +80,8 @@ QtObject {
             }
         ]
         filters: ValueFilter {
-            roleName: "walletType"
-            value: Constants.watchWalletType
-            inverted: true
+            roleName: "canSend"
```

Comment:
I'm wondering if this is needed in all the rest of stores that also work with `accounts`. Since right now we don't have the `accounts` model centralized in one unique place... Maybe you could take a look also at (not sure if I'm missing some other places):
- `..Profile/stores/WalletStore.qml`
- `SwapModelAdaptor.qml`

Reply:
I'll check

Reply:
Done. I updated 2 others.